CHANGELOG.md

```diff
@@ -20,7 +21,8 @@
 
 ## 1.0.0-rc.2 / 2020-08-09
 
-* Create https://toml.io as the new primary location to read the TOML spec.
+* Create [https://toml.io](https://toml.io) as the new primary location to read
+  the TOML spec.
```

Comment:
Does this belong here? It feels unrelated.

Reply:
Thhe markdown linter that I use is allergic to bare URLs, it seems. I'll undo that bit.